exercises-fold/reject/reject.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Iterates over `collection`, constructing and returning a new array containing
+ * the elements of `collection` for which `predicate` returns `false`.
+ *
+ * This is the opposite of `select`.
+ *
+ * @example
+ * reject([1, 2, 3, 4, 5, 6], isEven); // => [1, 3, 5]
+ *
+ * @param {any[]} collection - An array containing anything
+ * @param {function} predicate - A function that returns `true` or `false`
+ * @returns {array} A new array containing the elements of `collection`
+ *   for which `predicate` returns `false`
+ */
+function reject(collection, fn) {
```

Comment:
This pattern is at the heart of implementing `map`, too.

Reply:
Is this something that should be changed/improved, or just something to move forward with?  I don't understand how map would be implemented in the internal function within fold in this exercise...